content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -191,59 +216,46 @@ is found in `.git/refs/exps`, and earlier ones are in its [reflog].
 
 </details>
 
+Each experiment is derived from the <abbr>workspace</abbr> at the time it's
+queued. If you make changes in the workspace afterwards, they won't be reflected
+in queued experiments (once run).
+
+Run them all one-by-one with the `--run-all` flag. For isolation, this is done
+outside your <abbr>workspace</abbr> (in temporary directories).
```

Comment:
UPDATE: The Running Experiments has been revised again to consolidate into fewer sections and to remove non-experiment-related info. as well as lesser details (like the `--jobs` option which details are now in the cmd ref. only).

> Per https://github.com/iterative/dvc.org/pull/3182#issuecomment-1012558741 as well

Reply:
p.s. @iesahin I'm interested on your review of the latest changes to the guide as well. I deleted a good portion of the text you added but I think it can be reused in future guides e.g. for #2883 (will add a note there)